controllers/nginx/rootfs/etc/nginx/lua/vendor/lua-resty-http/lua-resty-http-0.09-0.rockspec

```diff
@@ -0,0 +1,22 @@
+package = "lua-resty-http"
```

Comment:
sure. The lua dependencies are required when you enable [custom errors](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/nginx/#custom-errors) (is not possible to create a request in nginx)